chrome_manifest_v3/class_model.js

```
@@ -0,0 +1,456 @@
+        organization_fields: 'displayName',
+        filter: 'open',
+        fields: 'name' /* "name,closed" */,
+      },
+      this.loadTrelloData_success_boards.bind(this),
+      this.loadTrelloData_failure.bind(this)
+    );
+    this.checkTrelloDataReady();
+  }
+
+  loadTrelloData_success_boards(data) {
+    let validData = Array();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'validData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)